aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
+
+The [`google.type.Interval`][interval] type (in the common protos described in
+[AIP-213][]) represents a range between two timestamps, with an inclusive start
+value and exclusive end value as described above.
+
+Ranges between two timestamps which conform to the expectations of the
+`Interval` message **should** usually use this rather than having separate
```

Comment:
Nit: I'd remove the "usually", the "should" gives all the lee-way one needs to _not_ use it.

Reply:
To me, it's a "**could** use". I *might* add "**could**" as another imperative in the site-generator. wdyt?

Reply:
I don't think "may" and "could" are sufficiently different. But I would support finding another standard term for "weak should" - i.e. "by default, go this way - but you don't need a strong reason to justify going the other way".